terminus-ui/input/src/input.component.ts

```diff
@@ -882,13 +880,30 @@ export class TsInputComponent implements
 
 
   /**
-   * Begin monitoring for the input autofill
+   * After the view is initialized, trigger any needed animations
    */
-  public ngOnInit(): void {
-    this.autofillMonitor.monitor(this.elementRef.nativeElement).subscribe(event => {
+  public ngAfterViewInit(): void {
+    // Begin monitoring for the input autofill
+    this.autofillMonitor.monitor(this.inputElement.nativeElement).subscribe(event => {
       this.autofilled = event.isAutofilled;
       this.stateChanges.next();
```

Comment:
This is probably the meat of this fix...

Reply:
Yep. Mainly we were passing a reference to our component rather than the actual `<input>` element. Moving the setup into `ngAfterViewInit` just makes sure all the `ngIf`s are settled before using that reference.